### PR TITLE
feat(stream-model-instance): Enforce that model field must refer to a Model StreamID

### PR DESCRIPTION
**Pull request by stbrody**

### CODE REVIEW

File: packages/stream-model-instance-handler/src/model-instance-document-handler.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import jsonpatch from 'fast-json-patch'
 import cloneDeep from 'lodash.clonedeep'
+import { Model } from '@ceramicnetwork/stream-model'
```
Comment (oed) [inline packages/stream-model-instance-handler/src/model-instance-document-handler.ts:3]: Please use the [StreamID](https://github.com/ceramicnetwork/js-ceramic/blob/develop/packages/streamid/src/stream-type.ts) package instead!

**Review by oed — CHANGES_REQUESTED** [review]

File: packages/stream-model-instance-handler/src/model-instance-document-handler.ts (hunk 1 above)
Comment (stbrody) [reply packages/stream-model-instance-handler/src/model-instance-document-handler.ts:3]: that doesn't give a way to directly get the Model streamtype code.  I'd either need to hardcode the type number here or hardcode the string description (`"model"`) and look up the code from that.  In either case this package would need to be making an assumption about the identifier of the Model streamtype, so at that point I might as well just hardcode the type number.  I can do that if you prefer.

Comment (oed) [reply packages/stream-model-instance-handler/src/model-instance-document-handler.ts:3]: Ok, hardcoding seems better imo. Type int should never change anyway.

Comment (stbrody) [reply packages/stream-model-instance-handler/src/model-instance-document-handler.ts]: updated

**Pull request merged**